src/async/pingfilter.rs

```diff
@@ -0,0 +1,109 @@
+use super::imports::*;
+use websocket::{OwnedMessage, WebSocketError};
+
```

Comment:
Not sure. I included it mostly for completeness. If discord never actually sends WS-level pings it might not matter.